packages/babel-plugin-transform-modules-systemjs/src/index.ts

```diff
@@ -436,6 +436,7 @@ export default declare<PluginState>((api, options: Options) => {
                 }
                 removedPaths.push(path);
               } else {
+                // @ts-expect-error TSDeclareFunction is not expected here
                 path.replaceWith(buildExportCall("default", declar));
```

Comment:
from my understanding `TSDeclareFunction` is not expected to get there(I would assume it is to be removed somewhere earlier, and it is fine to ignore this)